docs/connector-development/config-based/tutorial/5-incremental-reads.md

```diff
@@ -0,0 +1,262 @@
+      description: >-
+        Your API Access Key. See <a
+        href="https://exchangeratesapi.io/documentation/">here</a>. The key is
+        case sensitive.
+      airbyte_secret: true
+    base:
+      type: string
+      description: >-
+        ISO reference currency. See <a
+        href="https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html">here</a>.
+      examples:
+        - EUR
+        - USD
+```
+
+Then we'll set the `start_date` to last week our connection config in `secrets/config.json`.
+The following `echo` command will update your config with a start date set at 7 days prior to today.
+
+```
+echo "{\"access_key\": \"<your_access_key>\", \"start_date\": \"$(date -v -7d '+%Y-%m-%d')\", \"base\": \"USD\"}"  > secrets/config.json
+```
+
+And we'll update the `path` in the connector definition to point to `/{{ config.start_date }}`.
+Note that we are setting a default value because the `check` operation does not know the `start_date`. We'll default to hitting `/latest`:
+
+```
+retriever:
+  requester:
+      path:
+        type: "InterpolatedString"
+        string: "{{ config.start_date }}"
+        default: "/latest"
+```
+
+You can test the connector by executing the `read` operation:
+
+```python main.py read --config secrets/config.json --catalog integration_tests/configured_catalog.json```
+
+By reading the output record, you should see that we read historical data instead of the latest exchange rate.
+For example:
+> "historical": true, "base": "USD", "date": "2022-07-18"
+
+The connector will now always read data for the start date, which is not exactly what we want.
+Instead, we would like to iterate over all the dates between the start_date and today and read data for each day.
+
+We can do this by adding a `DatetimeStreamSlicer` to the connector definition, and update the `path` to point to the stream_slice's `start_date`:
+More details on the stream slicers can be found [here](./link-to-stream-slicers.md) <FIXME: need to fix links>
+
+Let's first define a stream slicer at the top level of the connector definition:
+
+```
+stream_slicer:
+  type: "DatetimeStreamSlicer"
+  start_datetime:
+    datetime: "{{ config.start_date }}"
+    datetime_format: "%Y-%m-%d"
+  end_datetime:
+    datetime: "{{ now_local() }}"
+    datetime_format: "%Y-%m-%d %H:%M:%S.%f"
+  step: "1d"
+  datetime_format: "%Y-%m-%d"
+```
+
+and refer to it in the stream's retriever. Note that we're also setting the `cursor_field` in the stream's `options` because it is used both by the `Stream` and the `StreamSlicer`:
+
+```
+rates_stream:
+  type: DeclarativeStream
+  options:
+    name: "rates"
+    cursor_field: "date
+  primary_key: "date"
+  schema_loader:
+    ref: "*ref(schema_loader)"
+  retriever:
+    ref: "*ref(retriever)"
+    stream_slicer:
+      ref: "*ref(stream_slicer)"
+```
+
+And we'll update the path to point to the `stream_slice`'s start_date
+
+```
+requester:
```

Comment:
I think this is the stream requester, but reading your example I was replacing the root requester.